webapp-django/crashstats/cron/management/commands/cronrun.py

```diff
@@ -66,10 +66,21 @@ def handle(self, **options):
         else:
             return self.cmd_run_all()
 
+    @contextlib.contextmanager
+    def stdout_to_logger(self, cmd):
+        class StdoutLogger:
+            def write(self, txt):
+                logger.info('%s: %s' % (cmd, txt.strip()))
+
+        stdout = self.stdout
+        self.stdout = StdoutLogger()
+        yield
+        self.stdout = stdout
```

Comment:
We want logging output when running on the servers which outputs mozlog format which goes on a trip to the Azores or something like that. However, Django commands don't use logging output--they use `self.stdout`, so we wrap that in a context when running jobs and magic happens.